packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -508,17 +506,27 @@ const getEchartAxis = ({
         axisItem: Field | TableCalculation | undefined,
     ) => {
         const field =
-            axisItem && getItemId(axisItem) && formats?.[getItemId(axisItem)];
-        return (
-            field &&
-            (field.format || field.round) && {
-                axisLabel: {
-                    formatter: (value: any) => {
-                        return formatValue(field.format, field.round, value);
-                    },
+            axisItem &&
+            getItemId(axisItem) &&
+            items.find((item) => getItemId(axisItem) === getItemId(item));
+        const hasFormatOrRound =
+            isField(field) && (field.format || field.round);
+        const axisMinInterval =
+            isDimension(field) &&
+            field.timeInterval &&
```

Comment:
should we check if field type is DATE ?

Reply:
I want to abstract all the logic into the config file. When we add new time intervals, ideally we would just need to edit that file.